test/support/test_index_helpers.rb

```diff
@@ -1,6 +1,6 @@
 class TestIndexHelpers
   AUXILIARY_INDEX_NAMES = ["page-traffic_test", "metasearch_test"].freeze
-  INDEX_NAMES = %w(mainstream_test government_test).freeze
+  INDEX_NAMES = %w(mainstream_test government_test govuk_test).freeze
```

Comment:
Again, we should be considering `govuk` as a separate thing from the existing index list - I think.